LUI.md

```diff
@@ -107,35 +109,41 @@ The `componentConfig` object can take in key value pairs of component names to n
 The `style` object can be used to override the predetermined style properties of the given component type.
 
 For example:
+
 ```js
 componentConfig: {
   ProgressBar: {
-    style: { radius: 20 }
+    style: {
+      radius: 20;
```

Comment:
nit (since this and the other js snippets are just examples) but `;` wouldn't be used to separate properties in the js theme file. this might've been the formatter adding these or something
```suggestion
      radius: 20
```

Reply:
Not sure why my linter made that change 😕 Thanks for catching it!